docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -242,47 +271,15 @@ It should be tested primarily in isolation as other components shouldn’t affec
 
 ### Graduation Criteria [optional]
 
-Alpha:
-
-The feature is initially constructed and toggleable with an environment variable
+Experimental:
 
-Beta:
-
-The feature has e2e tests implemented and is more integrated and seamless with the project.
+The feature is initially constructed and toggleable with an environment variable. It will be implemented as described in this document and have e2e tests implemented and integrated in the project.
 
 Stable: 
 
 The feature has been used for a while and is widely acceptable as well as reliable and will now be enabled by default.
 
-For AzureMachineTemplate we will move the feature enabled specification from metadata.annotations to a new optional field under spec.template.spec called nodeCachingMode which will be enabled if present and a sub field under that called cacheInterval which will be set to 1 day by default, and it will be a required subfield of nodeCachingMode if that field is there. For AzureMachinePool we will do the same thing under spec.template in the AzureMachinePool files.
-
-Example AzureMachineTemplate yaml:
-```yaml
-apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-kind: AzureMachineTemplate
-metadata:
-  name: node-os-image-caching-machine-template
-  namespace: default
-spec:
-  template:
-    spec:
-      nodeCachingMode:
-        interval: 24h
-```
-
-Example AzureMachinePool yaml:
-```yaml
-apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-kind: AzureMachinePool
-metadata:
-  name: node-os-image-caching-machine-pool
-  namespace: default
-spec:
-  template:
-    nodeCachingMode:
-      interval: 24h
-```
-
+At this point a more sophisticated method of choosing a healthy node will be used, preferrably by annotating the AzureMachine and AzureMachinePoolMachine instances after every patch. This will allow for more optimization and fewer unnecessary uses of this operation as if no updates in the interval are needed we would be able to now properly know.
```

Comment:
```suggestion
At this point a more sophisticated method of choosing a healthy node will be used, preferably by annotating the AzureMachine and AzureMachinePoolMachine instances after every patch. This will allow for more optimization and fewer unnecessary uses of this operation as if no updates in the interval are needed we would be able to now properly know.
```